masterlist.yaml

```diff
@@ -24397,6 +24397,13 @@ plugins:
         condition: 'active("Falskaar - SkyTEST and Bruma Patch.esp")'
 
   ### JK's Skyrim Patches ###
+  # JK's Skyrim - Rob's Bug Fixes
+  - name: 'JKs[ _]Skyrim_.*_Patch\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/6289/'
+        name: 'JK''s Skyrim'
+    after: [ 'JKs Skyrim - Rob''s Bug Fixes.esp' ]
```

Comment:
This should be grouped with JK's Skyrim since they're from the same page. Also, the new subsection header isn't necessary.